discovery-provider/src/queries/get_trusted_notifier_discrepancies.py

```diff
@@ -210,9 +215,9 @@ def get_track_delist_discrepancies(redis: Redis):
             ).replace(tzinfo=timezone.utc)
             # Only run query every 12h
             if latest_check > datetime.now(timezone.utc) - timedelta(hours=12):
-                track_delist_discrepancies = redis.get(
-                    TRACK_DELIST_DISCREPANCIES_KEY
-                ).decode()
+                track_delist_discrepancies = redis.get(TRACK_DELIST_DISCREPANCIES_KEY)
+                if track_delist_discrepancies is not None:
+                    track_delist_discrepancies = track_delist_discrepancies.decode()
                 if track_delist_discrepancies != "[]":
```

Comment:
Should these checks also check if `None` then?

Reply:
It's ok to return None here, this check just prevents an error if it's null in redis. If it's none, we'll just return None and the health check resp will be `null`